hack/common.sh

```diff
@@ -55,7 +55,7 @@ LATEST_NOOBAA_DB_IMAGE="centos/postgresql-12-centos7"
 LATEST_CEPH_IMAGE="ceph/daemon-base:latest-pacific"
 LATEST_VOLUME_REPLICATION_OPERATOR_IMAGE="csiaddons/volumereplication-operator:v0.1.0"
 LATEST_ROOK_CSIADDONS_IMAGE="csiaddons/k8s-sidecar:v0.2.1"
-LATEST_ROOK_CSI_NFS_IMAGE="mcr.microsoft.com/k8s/csi/nfs-csi:v3.1.0"
+LATEST_ROOK_CSI_NFS_IMAGE="k8s.gcr.io/sig-storage/nfsplugin:v3.1.0"
```

Comment:
you need to run `make gen-latest-csv` as this needs to be updated in ocs-operator.clusterserviceversion.yaml